cachelib/allocator/MMTinyLFU.h

```diff
@@ -456,6 +444,40 @@ class MMTinyLFU {
       // Tiny and main cache iterators
       ListIterator tIter_;
       ListIterator mIter_;
+    };
+
+    class LockedIterator : public Iterator {
```

Comment:
@igchor: is this necessary? TinyLFU doesn't support combined locking. It seems like we can just rename Iterator to LockIterator and avoid introducing this new code here

Reply:
Yes, you are right, I forgot to revert these changes after realizing TinyLFU does not support combined locking. Done.